docs/PresentationalConstraints.md

```diff
@@ -9,5 +9,6 @@ Used by output serialisers where string output is required. `value` must be:
 
 * a string recognised by Java's `String.format` method
 * appropriate for the data type of `field`
+* is not `null`
```

Comment:
Hi, i think simply "not null" will read better.

Value must be is not null is a bit confusing for me.